src/main/java/net/sf/jabref/collab/ChangeScanner.java

```diff
@@ -344,16 +344,14 @@ private static BibEntry bestFit(EntrySorter old, EntrySorter neu, int index) {
     }
 
     private void scanPreamble(BibDatabase inMem1, BibDatabase onTmp, BibDatabase onDisk) {
-        String mem = inMem1.getPreamble();
-        String tmp = onTmp.getPreamble();
-        String disk = onDisk.getPreamble();
-        if (tmp == null) {
-            if ((disk != null) && !disk.isEmpty()) {
-                changes.add(new PreambleChange(mem, disk));
-            }
+        String mem = inMem1.getPreamble().orElse(null);
+        Optional<String> tmp = onTmp.getPreamble();
+        Optional<String> disk = onDisk.getPreamble();
+        if (!tmp.isPresent()) {
```

Comment:
Yeah, I can only guess as well. `disk` is quite clear, but the difference between `mem` and `tmp` is not obvious...